src/main/java/me/botsko/prism/commands/ParamsCommand.java

```diff
@@ -38,7 +38,7 @@ private void help(CommandSender sender) {
 				+ " Force a worldwide search, for lookups only (unless configured for rollbacks)."));
 		sender.sendMessage(
 				Prism.messenger.playerMsg(ChatColor.LIGHT_PURPLE + "r:<player|x,y,z>:[radius]" + ChatColor.WHITE
-						+ " Base the radius around another place, like r:[player]:20 or r:20,35,10:5 (x,y,z)"));
+						+ " Base the radius around another place, like r:player:20 or r:20,35,10:5 (x,y,z)"));

```

Comment:
this should still use <player> to indicate the sender needs to replace that variable

Reply:
Sorry markdown filtered my `<`

Reply:
ie use `<player>`

Reply:
I'd agree with that.. you know someone is literally going to type `r:player:20` :facepalm: